### DISCUSSION TITLE
怎么改头像啊，手机里点更换头像就没动静了

### DISCUSSION THREAD

**Discussion by qingjiang-neko** [share121/inter-knot#5262] [委托]
![Screenshot_2024-09-21-11-59-41-421_com android browser](https://github.com/user-attachments/assets/76397897-8ff0-4320-bdeb-29d85b35703a)
大概就像图片里那样，点了之后就不动了

**Comment by KELERSAMA** [comment]
打开浏览器的电脑模式即可发现更改头像按钮

**Reply by qingjiang-neko** [reply]
感谢🥰
